src/display/canvas.js

```diff
@@ -856,7 +856,10 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
 
       if (this.transparentCanvas) {
         this.ctx = this.compositeCtx;
+        this.ctx.save();
```

Comment:
oh, I understand this part. By looking at beginDrawing, will just `restore()` do the same thing?